### PR TITLE
Make this work on heroku

### PR DESCRIPTION
**Pull request by alanmoo**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -9,6 +9,7 @@ google-api-python-client==1.5.5
 httplib2==0.9.2
 oauth2client==4.0.0
 oauthlib==2.0.0
+psycopg2==2.6.2
```
Comment (Pomax) [inline requirements.txt]: I don't see any changes to the code above that adds in this dependency, where did it come from?

**Comment by Pomax** [comment]
This looks R+, but why is there a new dependency in the requirements.txt?

**Comment by alanmoo** [comment]
I believe heroku was not happy without that dependency

**Comment by Pomax** [comment]
Is there a log that explains why? If it passes now, feel free to merge it in, I'm just not familiar with django enough yet to go "oh yeah of course, dependency X, obviously we need that" =P

**Comment by alanmoo** [comment]
There's probably a long somewhere deep in heroku, but I don't know it's worth the effort to find it. In any case, I haven't added any dependencies recently, so *something* wanted it there.

**Pull request merged**